src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1269,12 +1269,7 @@ public Tizen.NUI.PropertyMap Placeholder
             }
         }
 
-        /// <summary>
-        /// The Ellipsis property.<br />
-        /// Enable or disable the ellipsis.<br />
-        /// </summary>
-        /// <since_tizen> 6.5 </since_tizen>
-        /// This will be released at Tizen.NET API Level 6.5, so currently this would be used as inhouse API.
+        /// This will be released at Tizen.NET API Level 9, so currently this would be used as inhouse API.
```

Comment:
Please add summary.
You just need to remove the since_tizen tag.

Reply:
done